### PR TITLE
docs: add blog deep dive 1.12.1

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

**Comment by adela-bytebase** [comment]
ah... just realize it's in Chinese, I will update the images later.

File: content/blog/1-12-1-new-features.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+description: The second release of the 🐰  year, the version number was upgraded from 1.12.0 to 1.12.1, and although it was a minor upgrade (since no schema changes were involved), it still included 230 code commits
+---
+
+The second release of the 🐰  year, the version number was upgraded from 1.12.0 to 1.12.1, and although it was a minor upgrade (since no schema changes were involved), it still included 230 code commits, so let's see what new features were included 👇
+
+![commits](/static/blog/1-12-1-new-features/commits.webp)
+
+## Single Sign-On (SSO)
+
+The first is support for Single Sign-On (SSO), which should come standard at the enterprise level. What I regret is that this feature was made too late. Because the previous preview version is still stumbling, our customers can't wait to go with it, it's a lesson.
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:16]:
```suggestion
Bytebase now supports Single Sign-On (SSO). As a hindsight, it was a bit late to add this feature. We released a preview version in the last release, and our customers can’t wait to try it out. Anyway, better late than never.
```

File: content/blog/1-12-1-new-features.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+![SSO](/static/blog/1-12-1-new-features/SSO.webp)
+
+## Google Cloud Spanner
+
+Then there is the support for Google Cloud Spanner. It's worth mentioning that Bytebase be is the only database change management tool on the market that supports Spanner.
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:22]:
```suggestion
We also extend our database engine support to include Google Cloud Spanner. AFAIK, Bytebase is the only database change management tool on the market that supports Spanner.
```

File: content/blog/1-12-1-new-features.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+![gcp](/static/blog/1-12-1-new-features/gcp.webp)
+
+## Onboarding Guide
+
+This is the second version of our onboarding guide. The previous one forced the user to complete the whole guide - users were required to add a database instance, and if that failed, a dialog box popped up asking whether to create a Sample Instance. Then you have to create a project and transfer the database to the project before you can start making any changes. The pre-configuration involves too many steps, so for the new version, an instance and a project are prepared. Once the user registers, he/she can immediately experience the integration of Bytebase's two core capabilities change and query.
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:28]:
```suggestion
We also redesigned our onboarding guide. The previous one forced the user to complete the whole guide - users were required to add a database instance, and if that failed, a dialog box popped up asking whether to create a Sample Instance. Then users have to create a project and transfer the database to the project before they can start making any changes. The pre-configuration involves too many steps. To solve this issue, the new version prepares an instance and a project in advance. Once the user registers, he/she can immediately experience the integration of Bytebase's two core capabilities, `change` and `query`.
```

File: content/blog/1-12-1-new-features.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+- From the change process, you can experience the whole review collaborative workflow and the pre-configured auto SQL review strategy; the change history after the change is completed.
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:30]:
```suggestion
- From the change process, you can experience the whole change review collaborative workflow, the pre-configured auto SQL review strategy, and see the change history after the change completes.
```

File: content/blog/1-12-1-new-features.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- Then you enter SQL Editor, also pre-configured with query statements and anonymized data, to experience the thrill of fast data query and also perceive the data security features of data anonymization.
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:31]:
```suggestion
- Then you enter SQL Editor, also pre-configured with query statements and anonymized data, to experience the data query and check the data anonymization in effect.
```

File: content/blog/1-12-1-new-features.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+![onboarding1](/static/blog/1-12-1-new-features/onboarding1.webp)
+![onboarding2](/static/blog/1-12-1-new-features/onboarding2.webp)
+![onboarding3](/static/blog/1-12-1-new-features/onboarding3.webp)
+![onboarding4](/static/blog/1-12-1-new-features/onboarding4.webp)
+![onboarding5](/static/blog/1-12-1-new-features/onboarding5.webp)
+![onboarding6](/static/blog/1-12-1-new-features/onboarding6.webp)
+![onboarding7](/static/blog/1-12-1-new-features/onboarding7.webp)
+![onboarding8](/static/blog/1-12-1-new-features/onboarding8.webp)
+![onboarding9](/static/blog/1-12-1-new-features/onboarding9.webp)
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:45]:
```suggestion
```

File: content/blog/1-12-1-new-features.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+Well, that's it for this update, there are a lot of UX optimizations in this version, as the features become more comprehensive, we also start to do UX upgrades for the existing features, this onboarding guide optimization is an example. We expect to upgrade the entire SQL Review 100+ rule configuration experience to the release in two weeks.
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:47]:
```suggestion
Well, that's it for this update, there are a lot of UX optimizations in this version, as the features become more comprehensive, we also start to upgrade our existing UX, this onboarding guide refresh is the beginning. Next, we plan to upgrade the entire SQL Review 100+ rule configuration experience.
```

File: content/blog/1-12-1-new-features.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+## Partnership Announcement
+
+In addition, we also announced a technical partnership with GitHub and PingCAP for this release cycle, and will continue to build product integrations with more talented peers in the industry.
+
+Bytebase has a good position to integrate with all parties to create a 1 + 1 > 2 effect. This week ClickHouse CTO also came to our GitHub and contributed a PR, I'll contribute a PR to them someday.
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:53]:
```suggestion
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/blog/1-12-1-new-features.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+- From the change process, you can experience the whole change review collaborative workflow, the pre-configured auto SQL review strategy, and see the change history after the change completes.
+- Then you enter SQL Editor, also pre-configured with query statements and anonymized data, to experience the data query and check the data anonymization in effect.
+
+
+Well, that's it for this update, there are a lot of UX optimizations in this version, as the features become more comprehensive, we also start to upgrade our existing UX, this onboarding guide refresh is the beginning. Next, we plan to upgrade the entire SQL Review 100+ rule configuration experience.
+
+## Partnership Announcement
+
+In addition, we also announced a technical partnership with GitHub and PingCAP for this release cycle, and will continue to build product integrations with more talented peers in the industry.
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:38]:
```suggestion
In addition, we also announced technical partnership with [GitHub](/blog/bytebase-github-technology-partner) and [PingCAP](/blog/streamline-database-change-management-for-tidb-cloud-with-bytebase) during this release cycle, and will continue to build product integrations with more talented peers in the industry.
```

File: content/blog/1-12-1-new-features.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+![gh-ch](/static/blog/1-12-1-new-features/gh-ch.webp)
+![gh-bb](/static/blog/1-12-1-new-features/gh-bb.webp)
+![pingcap-bb](/static/blog/1-12-1-new-features/pingcap-bb.webp)
```
Comment (tianzhou) [inline content/blog/1-12-1-new-features.md:42]:
```suggestion
![gh-bb](/static/blog/1-12-1-new-features/gh-bb.webp)
![pingcap-bb](/static/blog/1-12-1-new-features/pingcap-bb.webp)
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**